### PR TITLE
mtcnn-feedstock

### PR DESCRIPTION
**Pull request by Tata17**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/mtcnn/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 56624f08e088e5cf062de8005df09696cdbc0f4c2fc3578483b19e0fb3984252
+
+build:
+  skip: True  # [py<34]
```
Comment (marcelotrevisani) [inline recipes/mtcnn/meta.yaml]: I think this recipe could be ``noarch``
```suggestion
  noarch: python
```

File: recipes/mtcnn/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+requirements:
+  host:
+    - keras >=2.0.0
+    - opencv >=4.1
+    - pip
+    - python
+  run:
+    - keras >=2.0.0
+    - opencv >=4.1
+    - python
```
Comment (marcelotrevisani) [inline recipes/mtcnn/meta.yaml]:
```suggestion
    - python >=3.4
```

File: recipes/mtcnn/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  skip: True  # [py<34]
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - keras >=2.0.0
+    - opencv >=4.1
+    - pip
+    - python
```
Comment (marcelotrevisani) [inline recipes/mtcnn/meta.yaml]:
```suggestion
    - python >=3.4
```

File: recipes/mtcnn/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  license_file: LICENSE.txt
+  summary: "Multi-task Cascaded Convolutional Neural Networks for Face Detection, based on TensorFlow"
+  dev_url: http://github.com/ipazc/mtcnn
+
+extra:
+  recipe-maintainers:
+    - Tata17
+    - oblute
+    - benhuff
+    - rluria14
```
Comment (marcelotrevisani) [inline recipes/mtcnn/meta.yaml]: Could you please confirm that you would like to comaintain this recipe?
@oblute @benhuff @rluria14

File: recipes/mtcnn/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    - opencv >=4.1
+    - python
+
+test:
+  imports:
+    - mtcnn
+    - mtcnn.exceptions
+    - mtcnn.network
+  requires:
+    - nose
```
Comment (marcelotrevisani) [inline recipes/mtcnn/meta.yaml]: as you are just importing the modules this ``requires`` is not necessary then.
Could you please remove it?

File: recipes/mtcnn/meta.yaml (hunk 4 above)
Comment (rluria14) [reply recipes/mtcnn/meta.yaml]: I will be a maintainer

File: recipes/mtcnn/meta.yaml (hunk 5 above)
Comment (Tata17) [reply recipes/mtcnn/meta.yaml]: Hi Marcelo, thanks for the help, requires is gone.

File: recipes/mtcnn/meta.yaml (hunk 4 above)
Comment (oblute) [reply recipes/mtcnn/meta.yaml]: I will also be a maintainer

Comment (benhuff) [reply recipes/mtcnn/meta.yaml]: i will also also be a maintainer

**Pull request merged**